packages/client/hmi-client/src/components/model/petrinet/tera-parameter-entry.vue

```diff
@@ -38,18 +35,11 @@
 					"
 				/>
 				<!-- Uniform Distribution -->
-				<template
-					v-if="
-						getParameterDistribution(modelConfiguration, parameterId).type ===
-						DistributionType.Uniform
-					"
-				>
+				<template v-if="distribution.type === DistributionType.Uniform">
```

Comment:
Should those be their own component?

Reply:
as discussed might be better when we support more than 2 distribution types 😄